lib/failover_ip.rb

```diff
@@ -70,18 +78,25 @@ def switch_ips
 
       old_target = current_target
 
-      Hooks.run_before failover_ip, old_target, new_ip[:target]
+      Hooks.run_before failover_ip, old_target, new_ip[:target], dry
 
-      raise unless HTTParty.post("#{base_url}/failover/#{failover_ip}", :body => { :active_server_ip => new_ip[:target] }, :basic_auth => basic_auth).success?
+      if !dry
+        response = HTTParty.post("#{base_url}/failover/#{failover_ip}", :body => { :active_server_ip => new_ip[:target] }, :basic_auth => basic_auth)
```

Comment:
here is the other location waiting for the logic switch to positive instead of negative if. maybe forgot about it.

```
if dry
  # log
else
  # do things
end
```

Reply:
-> 7e1e7a1